docs/guides/troubleshooting.md

```diff
@@ -1,9 +1,58 @@
-# Troubleshooting SDA services
+# Troubleshooting
 
-> TODO:
-> This guide is a stub and has yet to be finished.
-> If you have feedback to give on the content you would like to see, please contact us on
-> [github](https://github.com/neicnordic/neic-sda)!
+TODO:
+This guide is a stub and has yet to be finished.
+If you have feedback to give on the content you would like to see, please contact us on
+[github](https://github.com/neicnordic/neic-sda)!
 
-Exactly what's happened when a service is acting up can be hard to pinpoint.
 In this guide we aim to give some general tips on how to troubleshoot and restore services to working order.
+
+## After deployment checklist
+
+After having deployed the SDA services in a Federated setup, the following steps can be followed to ensure that everything is up and running correctly.
+
+### Services running
+
+The first step is to verify that the services are up and running and the credentials are valid. Make sure that,
+
+- credentials for access to RabbitMQ and Postgres are securely injected to the respective services in the form of secrets
+- all the pods/containers are in `Ready`/`Up` status.
+
+Next step is to make sure that the remote connections (CEGA RabbitMQ) are working. Login to the RabbitMQ admin page and check that,
+
+- the Federation status of the Admin tab is in state `running`
+- the Shovel status of the Admin tab is in state `running` for all 5 shovels.
+
+## End-to-end testing
+
+NOTE: This guide assumes that there exists a test instance account with Central EGA. Make sure that the account is approved and added to the submitters group.
+
+### Upload file(s)
+
+Upload one or a number of files of different sizes and check that,
+
+- the file(s) exists in the configured `inbox` of the storage backend (e.g. S3 bucket or POSIX path)
+- the file(s) entry exists in the database in the `sda.files` and `sda.file_event_log` tables
+- If the `s3inbox` is used, there should be an `uploaded` event for each specific file in the `sda.file_event_log`
+- the file(s) exists in the CEGA metadata portal (here for the test instance) under the tab files which can be accessed after pressing the three lines button.
```

Comment:
"...CEGA Submission portal (...) Files listing, which can be accessed after pressing the three lines menu button."